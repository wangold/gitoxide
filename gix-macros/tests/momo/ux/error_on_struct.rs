fn main() {}

/// Only functions work with momo
#[gix_macros::momo]
struct S;
