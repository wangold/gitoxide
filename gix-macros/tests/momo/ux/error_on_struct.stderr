error: expect a function
 --> tests/momo/ux/error_on_struct.rs:3:1
  |
3 | /// Only functions work with momo
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
